sdk/monitor/azure-monitor-ingestion/src/main/java/module-info.java

```diff
@@ -1,11 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
 
-module com.azure.monitor.ingestion {
+module com.azure.monitor.ingestion.implementation {
     requires transitive com.azure.core;
-
-    exports com.azure.monitor.ingestion;
-    exports com.azure.monitor.ingestion.models;
-
-    opens com.azure.monitor.ingestion to com.azure.core, com.fasterxml.jackson.databind;
+    requires transitive com.azure.json;
+    exports com.azure.monitor.ingestion.implementation;
```

Comment:
Why are we exporting the implementation package and not any of the public packages?